Makefile

```diff
@@ -9,7 +9,9 @@ run:
 
 create-migration:
 	@read -p "migration name: " name; \
-	migrate create -ext sql -dir internal/database/postgres/migration -seq $$name
```

Comment:
the migration command should be something like `migrate create -ext sql -dir internal/constant/query/schemas -tz "UTC" $$name` (although I did not check if it will work correctly). 

Also, there are few issues with timestamps I found in the wild, that might be related and need checking

https://github.com/golang-migrate/migrate/issues/1024
https://github.com/golang-migrate/migrate/issues/988

Reply:
thank you, about issues not sure, but it is working well